package.json

```diff
@@ -47,6 +47,11 @@
 				"category": "AutoLISP",
 				"title": "%autolispext.loadcurrentfile.title%"
 			},
+			{
+				"command": "autolisp.openWebHelp",
+				"category": "AutoLISP",
+				"title": "Open Web Help"
```

Comment:
it needs to be put in the same file as above "%autolispext.loadcurrentfile.title%", which needs to be localized for different languages.

Reply:
Okay, I see the package.nls.json that is the default and the various other language json files. Should be an interesting time with a translator to generate those.

My thought is to generate it once with all 3 words and convert them back. If they are 1-1, then they are fine, If they are not the same as the original input, then maybe try again with each individual word as input to find the disconnect.

Reply:
I believe I have all the language files (added 1 line to each) updated and the package.json using the pointer, but I ran into the same problem of VSCode refactoring the spacing of the indentations on json files. Is this some rogue extension I am using for basic HTML editing? Are any of these known to cause this problem?
abusaidm.html-snippets
alefragnani.Bookmarks
eamodio.gitlens
ecmel.vscode-html-css
eg2.tslint
eg2.vscode-npm-script
felixfbecker.php-debug
GitHub.vscode-pull-request-github
lonefy.vscode-JS-CSS-HTML-formatter
ms-mssql.mssql
ms-python.python
ms-vscode-remote.remote-containers
ms-vscode-remote.remote-ssh
ms-vscode-remote.remote-ssh-edit
ms-vscode-remote.remote-wsl
ms-vscode-remote.vscode-remote-extensionpack
mtxr.sqltools
mtxr.sqltools-driver-sqlite
pranaygp.vscode-css-peek
ritwickdey.LiveServer
sidthesloth.html5-boilerplate
sneezry.watch-in-chrome
Zignd.html-css-class-completion

Reply:
I found the JSON formatter option in the settings, I'll revert that commit and see if I that fixes the issue I've been having.

Reply:
For future reference this is what was causing the problem and why no settings I applied seemed to prevent json files from converting tabs to spaces on save.
**lonefy.vscode-JS-CSS-HTML-formatter**

I still have other problems regarding the lines I am editing, but at least it won't flag the whole file as a change now.